src/promptflow-evals/tests/recordings/azure/test_metrics_upload_TestMetricsUpload_test_writing_to_run_history.yaml

```diff
@@ -109,34 +110,34 @@ interactions:
-        "token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjY0NDBENzEwMkY3OEJENkVDNDMyRURBNjk4QTQyRjM1MjU0RDc2NDgiLCJ0eXAiOiJKV1QifQ.eyJyb2xlIjoiQ29udHJpYnV0b3IiLCJzY29wZSI6Ii9zdWJzY3JpcHRpb25zL2IxNzI1M2ZhLWYzMjctNDJkNi05Njg2LWYzZTU1M2UyNDc2My9yZXNvdXJjZUdyb3Vwcy9uaXJvdmlucy1yZy1lYXN0dXMvcHJvdmlkZXJzL01pY3Jvc29mdC5NYWNoaW5lTGVhcm5pbmdTZXJ2aWNlcy93b3Jrc3BhY2VzL25pcm92aW5zLWFpLXByb2plY3QiLCJhY2NvdW50aWQiOiIwMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJ3b3Jrc3BhY2VJZCI6ImJkY2QxYTg3LWUwODAtNDc2Yi04NjE3LTQ3NDkxNjU4NDNiYSIsInByb2plY3RpZCI6IjAwMDAwMDAwLTAwMDAtMDAwMC0wMDAwLTAwMDAwMDAwMDAwMCIsImRpc2NvdmVyeSI6InVyaTovL2Rpc2NvdmVyeXVyaS8iLCJ0aWQiOiI3MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDciLCJvaWQiOiI3ODc1YmFhNi0wMjhkLTQ5YzgtYmQ0ZC1hNzcyYWE0ZGMwMTMiLCJwdWlkIjoiMTAwM0JGRkRBOEE0RDBFNyIsImlzcyI6ImF6dXJlbWwiLCJhcHBpZCI6Ik5pa29sYXkgUm92aW5za2l5IiwiZXhwIjoxNzIwNTQ5MjcyLCJhdWQiOiJhenVyZW1sIn0.Q1vf5F-TYKx8pzdaSXXtdMqiyc2SpGh24aRTxh8V4-LeGIlcWrUrTd3lJhcyuU0aj5Rxt_rx_bCKqS_tCx8_LO0Ss_N8WTLIKR2LYMWfnPVr08JSWBr7Xbq-SHamiL_Czf-lsNGfxnUPUYOH7VM9AOPst4hlAEE-P5TpftHFZDQSOWQx5VD9LV_e6NN_3Td0d8BR-iUrBVhKcU0LyeYsKCc-aYf6c2UdBvdwl16tf8yU_QD2OOurX6TWRMTTt5nS6ebL8u-BDiJlD4cu9pXKjc6gydY7JsYF0xRzFqDaet3gPhnrnSDwLRWx7XdUmTDpiNI9cJ4A9Eet_csSJlFbLg",
-        "tokenExpiryTimeUtc": "2024-07-09T18:21:12.362991+00:00", "error": null, "warnings":
-        null, "revision": 2, "statusRevision": 0, "runUuid": "a5390638-bc06-4129-b24f-2a96292f0df3",
-        "parentRunUuid": null, "rootRunUuid": "a5390638-bc06-4129-b24f-2a96292f0df3",
-        "lastStartTimeUtc": "2024-06-18T18:21:00.126+00:00", "currentComputeTime":
-        "00:00:00", "computeDuration": null, "effectiveStartTimeUtc": "2024-06-18T18:21:00.126+00:00",
+        "token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IkZBMTRBODAyQjhDRTQ0MjQxRjE0NkU1RDgyNUQwNzJDODE3NjM0ODQiLCJ0eXAiOiJKV1QifQ.eyJyb2xlIjoiQ29udHJpYnV0b3IiLCJzY29wZSI6Ii9zdWJzY3JpcHRpb25zL2IxNzI1M2ZhLWYzMjctNDJkNi05Njg2LWYzZTU1M2UyNDc2My9yZXNvdXJjZUdyb3Vwcy9uaXJvdmlucy1yZy1lYXN0dXMvcHJvdmlkZXJzL01pY3Jvc29mdC5NYWNoaW5lTGVhcm5pbmdTZXJ2aWNlcy93b3Jrc3BhY2VzL25pcm92aW5zLWFpLXByb2plY3QiLCJhY2NvdW50aWQiOiIwMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJ3b3Jrc3BhY2VJZCI6ImY2YTcyYTc0LWY1NWEtNGRiOS04NmNmLTBiNTc3ZTZkYWQzZCIsInByb2plY3RpZCI6IjAwMDAwMDAwLTAwMDAtMDAwMC0wMDAwLTAwMDAwMDAwMDAwMCIsImRpc2NvdmVyeSI6InVyaTovL2Rpc2NvdmVyeXVyaS8iLCJ0aWQiOiI3MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDciLCJvaWQiOiI3ODc1YmFhNi0wMjhkLTQ5YzgtYmQ0ZC1hNzcyYWE0ZGMwMTMiLCJwdWlkIjoiMTAwM0JGRkRBOEE0RDBFNyIsImlzcyI6ImF6dXJlbWwiLCJhcHBpZCI6Ik5pa29sYXkgUm92aW5za2l5IiwiZXhwIjoxNzIxMzUwNjYyLCJhdWQiOiJhenVyZW1sIn0.STZiicI55Jz8o0PSqzP3641BU7wgxpe5z68nnRlQbtE2UNEu_MawQl156nHHFe8On7h-_9T2aphGlou_iEO1dQfG9TkckNV86Hbl-qwKB_vDusdd9FEM6u87EiXZYJ6m9YOqQBTs0DSzYaEOn30mW0lRXvdvb-bZCKYs4ICSkR_JMEkvs8qdHAIWxe9puMC8KGKm_VHVP0zuklsFwOw6XJ05CZ06H8mf9cN948kPKgDBlTXN9TMzS02qoVHra_Vnyt84QRMVBCAfdrLghc4BrmEN75S0o9upc2VeTaY_8JhXo51UQP3qBpRohlNRU0vx1Bv_kytu8uNPuq5FiXuEWQ",

```

Comment:
maybe it's better to sanitize this

Reply:
Fixed, I have also added sanitizer to remove names/aliases of developers.